service/src/test/java/uk/gov/hmcts/reform/fpl/service/MessageJudgeServiceTest.java

```diff
@@ -222,35 +235,49 @@ void shouldInitialiseJudicialMessagesWithOpenStatusWhenC2DocumentsDoNotExist() {
 
         Map<String, Object> expectedData = Map.of(
             "hasJudicialMessages", "Yes",
-            "judicialMessageDynamicList", expectedJudicialDynamicList
+            "judicialMessageDynamicList", expectedJudicialDynamicList,
+            "judicialMessageMetaData", JudicialMessageMetaData.builder()
+                .sender(COURT_EMAIL)
+                .recipient(EMPTY).build()
         );
 
         assertThat(data).isEqualTo(expectedData);
     }
 
     @Test
     void shouldReturnEmptyMapWhenC2DocumentsDoNotExist() {
```

Comment:
Test name is no longer returning empty map so it's worth updating test name.

Reply:
renamed the test

Reply:
Are you sure you did? I do not see it.

Reply:
sorry, it was missing earlier. it's updated now